sql/driver/http_post.go

```diff
@@ -27,7 +27,8 @@ import (
 	"net/http"
 	"time"
 
-	snappy "github.com/cockroachdb/c-snappy"
+	"github.com/golang/snappy"
```

Comment:
I think we could fairly easily make the usage of `c-snappy` optional depending on whether `cgo` is enabled using the `cgo` build tag.